Data4LifeSDKUtils.podspec

```diff
@@ -1,17 +1,20 @@
 Pod::Spec.new do |s|
   s.name             = "Data4LifeSDKUtils"
-  s.version          = "0.3.0"
+  s.version          = "0.4.0"
   s.summary          = "Generic utilities functions used in the D4L SDK libraries"
   s.homepage         = "https://github.com/d4l-data4life/d4l-utils-ios.git"
-  s.license          = 'Private License'
+  s.license          = { :type => 'LICENSE', :file => "LICENSE" }
   s.author           = { "D4L data4life gGmbH" => "contact@data4life.care" }
-  s.source           = { :git => "https://github.com/d4l-data4life/d4l-utils-ios.git", :tag => s.version }
-  s.swift_version    = '5.3'
 
+  s.source           = { :http => 'https://github.com/d4l-data4life/d4l-utils-ios/releases/download/0.4.0/XCFrameworks-0.4.0.zip' }
```

Comment:
Could we get this dynamically so we get always the current version using `s.version`?

Reply:
I did, seems to work